src/main/fc/cli.c

```diff
@@ -2253,64 +2253,95 @@ static int parseOutputIndex(char *pch, bool allowAllEscs) {
 
 #ifdef USE_DSHOT
 
-#define ESC_INFO_V1_EXPECTED_FRAME_SIZE 15
-#define ESC_INFO_V2_EXPECTED_FRAME_SIZE 21
+#define ESC_INFO_KISS_V1_EXPECTED_FRAME_SIZE 15
+#define ESC_INFO_KISS_V2_EXPECTED_FRAME_SIZE 21
+#define ESC_INFO_BLHELI32_EXPECTED_FRAME_SIZE 64
+
+enum {
+    ESC_INFO_KISS_V1,
+    ESC_INFO_KISS_V2,
+    ESC_INFO_BLHELI32
+};
 
 #define ESC_INFO_VERSION_POSITION 12
 
-void printEscInfo(const uint8_t *escInfoBytes, uint8_t bytesRead)
+void printEscInfo(uint8_t *escInfoBuffer, uint8_t bytesRead)
```

Comment:
Should this not be `void printEscInfo(const uint8_t *escInfoBuffer, uint8_t bytesRead)`?

Reply:
It _should_, yes. But to avoid having to copy the ESC identifer string for BLHeli32, I am overwriting the checksum after the end of the string with a string terminator: https://github.com/betaflight/betaflight/pull/3987/files#diff-72a6342f16b60f7944720f4dc08a71e1R2342
Any suggestions on how to do this better are appreciated.